dsf-gdb/org.eclipse.cdt.dsf.gdb/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.cdt.dsf.gdb;singleton:=true
-Bundle-Version: 7.0.0.qualifier
+Bundle-Version: 7.1.0.qualifier
```

Comment:
should it remain on 7.1 or it should be 7.0.100 ?

Reply:
7.1 because of new API method getBinary